From 4abdd28bb5ca4d7bf57b6ae3380e833a83d0cf5f Mon Sep 17 00:00:00 2001
From: Markus Pargmann <mpa@pengutronix.de>
Date: Sun, 7 Jul 2013 15:13:30 +0200
Subject: [PATCH 054/161] ARM: dts: imx27 phyCARD-S, move i2c1 and owire to
 rdk

Both buses are not used on the phyCARD-S module. This patch moves them
to the rdk file. Remove ioexpander.

Signed-off-by: Markus Pargmann <mpa@pengutronix.de>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27-phytec-phycard-s-rdk.dts |   13 +++++++++++++
 arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts |   18 ------------------
 2 files changed, 13 insertions(+), 18 deletions(-)

diff --git a/arch/arm/boot/dts/imx27-phytec-phycard-s-rdk.dts b/arch/arm/boot/dts/imx27-phytec-phycard-s-rdk.dts
index 390d8f2..0514b0a 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycard-s-rdk.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycard-s-rdk.dts
@@ -41,6 +41,19 @@
 	status = "okay";
 };
 
+&i2c1 {
+	status = "okay";
+
+	rtc@51 {
+		compatible = "nxp,pcf8563";
+		reg = <0x51>;
+	};
+};
+
+&owire {
+	status = "okay";
+};
+
 &sdhci2 {
 	cd-gpios = <&gpio3 29 0>;
 	status = "okay";
diff --git a/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts b/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
index cdff8c7..c8d57d1 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
@@ -33,20 +33,6 @@
 	status = "okay";
 };
 
-&i2c1 {
-	status = "okay";
-
-	ioexpander@41 {
-		compatible = "nxp,pca9536";
-		reg = <0x41>;
-	};
-
-	rtc@51 {
-		compatible = "nxp,pcf8563";
-		reg = <0x51>;
-	};
-};
-
 &i2c2 {
 	status = "okay";
 
@@ -56,7 +42,3 @@
 		reg = <0x52>;
 	};
 };
-
-&owire {
-	status = "okay";
-};
-- 
1.7.10.4

